README.md

```diff
@@ -1,11 +1,21 @@
-# DeFi Swap AI Assistant 
-<img src="https://i.imgur.com/xUkwJiK.png" alt="cover_image"  width="0" />
+# Ref Finance DeFi Swap Agent
+
+<img src="https://github.com/user-attachments/assets/aa54bac4-30ef-49bb-bac7-732ff561bd95" alt="cover_image"/>
 
 Ref Finance Agent is a template for creating a Bitte.ai Plugin for facilitating DeFi swaps using Ref Finance. Built using Next.js 14 + Elysia.
 
 [![Demo](https://img.shields.io/badge/Demo-Visit%20Demo-brightgreen)](https://ref-finance-agent-next.vercel.app/)
 [![Deploy](https://img.shields.io/badge/Deploy-on%20Vercel-blue)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2FMintbase%2Fref-finance-agent-next)
 
+## Demo
```

Comment:
This #Demo and ##Deploy should be after ##Project Walktrhough
i was refering to this session in my comment `[![Demo](https://img.shields.io/badge/Demo-Visit%20Demo-brightgreen)](https://ref-finance-agent-next.vercel.app/)
[![Deploy](https://img.shields.io/badge/Deploy-on%20Vercel-blue)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2FMintbase%2Fref-finance-agent-next)` sorry if i confused you.

Reply:
Makes sense, update.  Also removed width="0" from last `<img>` tag, unsure of why it was set that way.